paderbox/array/interval/core.py

```diff
@@ -514,16 +555,33 @@ def __getitem__(self, item):
         False
         >>> ai[29]
         True
-     
+        >>> ai[-25:-20]
+        array([ True,  True,  True,  True,  True])
+        >>> ai[-1:]
+        array([])
+
         """
         if isinstance(item, (int, np.integer)):
+            index = item
+            if index < 0:
+                index = index + self.shape[0]
```

Comment:
`.shape[0]` -> `.shape[-1]` Just for the case, we support independent dims in the furture

Reply:
Shape may be None. Can you catch this case?

Reply:
Done